app/schema.graphql

```diff
@@ -337,7 +337,7 @@ input DeleteApiKeyInput {
 }
 
 type DeleteApiKeyMutationPayload {
-  id: GlobalID!
```

Comment:
this stops a warning in the console about two different types having the same id field, the warning:

```
warning: RelayModernRecord: Invalid record update, expected both versions of record 
`U3lzdGVtQXBpS2V5OjQ=` to have the same `__typename` but got conflicting types 
`SystemApiKey` and `DeleteApiKeyMutationPayload`. 
The GraphQL server likely violated the globally unique id requirement by returning the
 same id for different objects.
```

Reply:
Hmm, intersting